docs/source/interfaces/Engineering_Diffraction.rst

```diff
@@ -29,6 +29,15 @@ Instrument
 Close
   Close the interface
 
+RB Number
+  To enable the GUI specify a RB Number
```

Comment:
I'd say a bit about what the RB number implies. I think that is not explained anywhere else in this doc.
Something like: "the RB Number will be used for the output paths so that files from different users and/or experiments can be kept separate", or similar.